libs/packages/components/src/lib/autocomplete-search/autocomplete-search.component.ts

```diff
@@ -302,6 +308,14 @@ export class SDSAutocompleteSearchComponent implements ControlValueAccessor {
         const val = this.createFreeTextItem();
         this.selectItem(val);
       }
+    } else if (
+      KeyHelper.is(KEYS.SPACE, event) &&
+      this.configuration.useCheckBoxes &&
+      this.highlightedIndex >= 0 &&
+      this.inputValue === ''
```

Comment:
Space functionality is not working when there filter applied for the results list